server/v1/controllers/shows/utils/utils__mongoose.js

```diff
@@ -162,12 +162,12 @@ function master_time_id(_id, start_time) {
 }
 
 function master_time_id__byShowType(show) {
-  if (show.master_show_uid) {
+  if (show.master_event_uid) {
```

Comment:
Needs to be changed to master_event_id (currently uid)

Reply:
thanks, got it fixed!